weatherman/weatherreport.py

```diff
@@ -0,0 +1,64 @@
+"""
+For reading data and generating weather report
+"""
+import os
+
+from utils import replace_string_in_list
+from weatheryearlysummary import WeatherYearlySummary
+
+
+class WeatherReport:
+
+    def __init__(self, path_to_dir):
+        self.__path_to_dir = path_to_dir
+        self.weather_data = dict()
+
+    def __should_ignore_reading(self, data):
+        for i in data:
+            if not i:
+                return True
+        else:
+            return False
+
+    def __parse_line_data(self, header, line):
+        return [line[header.index(attributes)] for attributes in WeatherYearlySummary.KEYS_WEATHER_ATTRIB]
```

Comment:
Why do we need to get index of key for parsing every line?
Shouldn't it be `attribute`?

Reply:
Heading in few files have different order. It needs to refrsh index on opening each file.